### PR TITLE
Feature/op 374 # Added Sylius 1.13 support 

### PR DESCRIPTION
**Pull request by tomkalon**
+ added Sylius 1.13 support
+ installation.md has been rebuild and fixed
+ Guzzle has been changed to HttpClient

### CODE REVIEW

File: src/Api/WebClient.php [hunk 1]
```diff
@@ -188,38 +200,48 @@ public function getAuthorizedHeaderWithContentType(): array
...
+            $request = $this->requestFactory
+                ->createRequest($method, $url)
+                ->withHeader('Content-Type', $header['Content-Type'])
+                ->withHeader('Authorization', $header['Authorization'])
+                ->withBody($this->streamFactory->createStream(json_encode($data)));
+
+            $result = $this->apiClient->sendRequest($request);
+            $response = json_decode((string) $result->getBody(), true);
+
+            if (201 < $result->getStatusCode()) {
```
Comment (senghe) [inline src/Api/WebClient.php:224]: The max number of 2xx HTTP is 203. To be honest, I don't understand this condition.

File: src/Api/WebClientInterface.php [hunk 2]
```diff
@@ -80,15 +80,10 @@ public function getAuthorizedHeaderWithContentType(): array;
...
-     *
-     * @throws GuzzleException
-     */
     public function request(
         string $method,
         string $url,
         array $data = [],
-        bool $returnJson = true
+        bool $returnJson = true,
     );
```
Comment (senghe) [inline src/Api/WebClientInterface.php:88]: Missing return type.

File: src/Resolver/OrganizationIdResolverInterface.php [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
 declare(strict_types=1);
 
 /*
```
Comment (senghe) [inline src/Resolver/OrganizationIdResolverInterface.php:12]: Could you check, why we have doubled OS header? And I would also ask you for checking whether it's the only place with it.

**Review by senghe — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -21,23 +21,23 @@ jobs:
...
+        sylius: [ "~1.12.0", "~1.13.0" ]
+        node: [ "^18.x", "20.x" ]
         mysql: [ "8.0" ]
-
+        
         exclude:
-          - sylius: ~1.11.0
-            symfony: "^6.0"
-
+          - sylius: "~1.13.0"
```
Comment (senghe) [inline .github/workflows/build.yml:31]: But you have no 8.0 in your matrix...

**Review by senghe — CHANGES_REQUESTED** [review]

File: src/Api/WebClient.php (hunk 1 above)
Comment (tomkalon) [reply src/Api/WebClient.php:224]: Fixed:
```
            if (200 !== $result->getStatusCode() && 201 !== $result->getStatusCode()) {
                throw new InvalidInPostResponseException();
            }
```
HHTP 200 - is returned when you select a parcel machine on the map
HTTP 201 - is returned when shipping export action is performed

File: src/Api/WebClientInterface.php (hunk 2 above)
Comment (tomkalon) [reply src/Api/WebClientInterface.php:88]: Fixed.
array|string
PHPStan didn't throw an error.

File: src/Resolver/OrganizationIdResolverInterface.php (hunk 3 above)
Comment (tomkalon) [reply src/Resolver/OrganizationIdResolverInterface.php:12]: OrganizationIdResolverInterface OS_Header double appearance has been fixed.

File: .github/workflows/build.yml (hunk 4 above)
Comment (tomkalon) [reply .github/workflows/build.yml:31]: Fixed.

**Review by senghe — APPROVED** [review]

**Pull request merged**